docker/Dockerfile-app

```diff
@@ -27,10 +27,6 @@ COPY . .
 
 # install poetry
 RUN curl -sSL https://install.python-poetry.org | python3 -
-# install specific version of pip due to installation issue with rpm-py-installer
-RUN pip install pip==23.0.1
-# install rpm-py-install separately as it does not conform with pep517
-RUN pip install rpm-py-installer==1.2.0 --no-use-pep517
 
 RUN poetry install --with server
```

Comment:
These Dockerfiles are also used in production so probably we should add the installation of rpm bindings conditionally.
E.g if on _dev_ or _test_ env than use rpmdyn; else use rpm-py-installer
Actually what will poetry install in this setup?  I guess it will also skip the rpmdyn because it's a _test_ dependency.

@rohanpm  Do you think we can already use `rpmdyn` in production? Or should we use it only for CI?

Reply:
You probably can, assuming you're only using a small part of the rpm library.

However if we're talking specifically about container images, where there is not really any need to use an isolated virtualenv, I think my preference usually would be to just use the OS package and not declare a dependency on *either* rpmdyn or rpm-py-installer in that case.  This is why, in other existing projects where rpmdyn or rpm-py-installer was used, it was only added into *test* requirements and not setup.py or requirements.txt.

I'm not all that experienced with poetry but I do notice above there's a POETRY_VIRTUALENVS_CREATE=false.  If this means it uses the OS's python environment then I guess installing the OS's python rpm bindings will work already if you find a way to skip rpmdyn & rpm-py-installer there.